internal/handlers/sqlite/msg_create.go

```diff
@@ -108,3 +119,29 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, lazyerrors.Error(err)
 	}
 }
+
+// validateCollectionNameRe validates collection names.
+// Empty collection name, names with `$` and `\x00` are not allowed.
+// Collection names that start with `.` are also not allowed.
```

Comment:
Collection name starting with `.` is checked by `strings.HasPrefix(collectionName, ".")` so we can simply this right?

Also let's mention valid collection name length :hugs:

Reply:
It's left here just to show what's the regex of proper collection name